packages/flame/lib/src/components/component.dart

```diff
@@ -18,11 +18,12 @@ import 'cache/value_cache.dart';
 /// called automatically once the component is added to the component tree in
 /// your game (with `game.add`).
 class Component with Loadable {
-  /// Whether this component is a HUD (Heads-up display) object or not.
+  /// Whether this component should respect the camera or not.
   ///
-  /// HUD objects ignore the FlameGame.camera when rendered (so their position
-  /// coordinates are considered relative to the device screen).
-  bool isHud = false;
+  /// Components that have this property set to false will ignore the
+  /// FlameGame.camera when rendered (so their position coordinates are
+  /// considered relative only to the viewport instead).
+  bool respectCamera = true;
```

Comment:
Maybe add a note here that the property is provisional, and we plan to remove it post-v1?

Reply:
I don't think that is necessary yet, we don't even have a plan for how to remove it?

Reply:
I would just say that it is only respected for top-level components, and we can think of whether we want to change that interface or not later